query/stdlib/influxdata/influxdb/from.go

```diff
@@ -20,10 +17,10 @@ type FromOpSpec struct {
 }
 
 func init() {
-	fromSignature := semantic.MustLookupBuiltinType("influxdata/influxdb", "from")
-	runtime.ReplacePackageValue("influxdata/influxdb", influxdb.FromKind, flux.MustValue(flux.FunctionValue(FromKind, createFromOpSpec, fromSignature)))
-	flux.RegisterOpSpec(FromKind, newFromOp)
-	plan.RegisterProcedureSpec(FromKind, newFromProcedure, FromKind)
+	// fromSignature := semantic.MustLookupBuiltinType("influxdata/influxdb", "from")
```

Comment:
I guess that the signature is the same in `flux` and `influxdb` now, so can you just delete this `init` method?

Reply:
Yes we can get rid of the replace stuff.